_posts/2017-10-28-soap-to-json-converter.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: Creating a SOAP to JSON converter service
+date: 2017-10-28
+---
+
+I recently had to consume a SOAP based web service from within JavaScript code. To solve that problem I decided to write a separate service acting as a SOAP to JSON converter. In this post I explain why and how I did it.
+<br/>
+If you are interested in the code you take a look to my GitHub repository at that [commit](https://github.com/caroleolivier/commute-status/tree/0b93d6b4e59c7183201f4a5a4cb5dade5fae73d7).
```

Comment:
"You can take a look _in_ my Github repository at _this_ commit"

Reply:
oh yeah it is better. It did sound odd, I changed the in, that, this quite a few times haha!